src/cmd/linuxkit/pkglib/build.go

```diff
@@ -250,7 +253,7 @@ func (p Pkg) Build(bos ...BuildOpt) error {
 					fmt.Fprintf(writer, "%s found or pulled\n", archRef)
 					skipBuild = true
 				} else {
-					fmt.Fprintf(writer, "%s not found\n", archRef)
+					fmt.Fprintf(writer, "%s not found: %s\n", archRef, err)
```

Comment:
This doesn't work. It is referring to the `err` but that is the one from `reference.Parse()`, which we already respond to ~10 lines above. the one from 3 lines above is scoped to that `if` statement.

Reply:
It will. Correct, err is scoped to `if` statement from `ImagePull`, and we are still inside that `if-else` here, so we will show an error from `ImagePull`.

Reply:
You are right.